pkg/v1/partial/uncompressed.go

```diff
@@ -81,6 +81,25 @@ func (ule *uncompressedLayerExtender) calcSizeHash() {
 	})
 }
 
+func (ule *uncompressedLayerExtender) Descriptor() (v1.Descriptor, error) {
```

Comment:
A comment explaining why we have to do this would be good.

Reply:
Done